panther_description/README.md

```diff
@@ -38,6 +38,10 @@ Parameter `wheel_config_path` allows using non-standard wheels with Panther robo
 - `mesh_package` - ROS package name to search for custom meshes. Used in evaluation **$(find my_amazing_package)/**.
 - `folder_path` - path used to search for mesh files within the ROS package.
 - `kinematics` - kinematics type. Possible options: `differential`, `mecanum`.
+- `odom_stderr` - standard deviation used to populated odometry message:
+  - `vel_x` - standard deviation for linear velocity in **x** direction.
+  - `vel_y` - standard deviation for linear velocity in **y** direction.
+  - `vel_yaw` - standard deviation for angular velocity alongside **z** axis.
```

Comment:
```suggestion
  - `vel_x` - standard deviation for linear velocity in **X** direction.
  - `vel_y` - standard deviation for linear velocity in **Y** direction.
  - `vel_yaw` - standard deviation for angular velocity alongside **Z** axis.
```
In other other docs axis are written as capital letters.

Reply:
Inertia has lower case letters for axes. Either this stays lower case or both are capital